staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1673,12 +1673,14 @@ message MemoryUsage {
     UInt64Value available_bytes = 3;
     // Total memory in use. This includes all memory regardless of when it was accessed.
     UInt64Value usage_bytes = 4;
+    // Total commit memory in use. Commit memory is total of physical and virtual memory in use.
+    UInt64Value commit_memory_bytes = 5;
```

Comment:
To be backwards compatible, the change needs to be appended. see the rules: https://protobuf.dev/getting-started/pythontutorial/#extending-a-protobuf